wiki/en/Running-a-Server.md

```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
+
+Replace `genreServer:port` in the example above with one of the following options:
+
+
+| Genre |   Server address           |
+|-----------|------------------|
+|**Any Genre 1** |`anygenre1.jamulus.io:22124`|
+|**Any Genre 2** |`anygenre2.jamulus.io:22224`|
+|**Any Genre 3** |`anygenre3.jamulus.io:22624`|
+|**Genre Rock** |`rock.jamulus.io:22424`|
+|**Genre Jazz** |`jazz.jamulus.io:22324`|
+|**Genre Classical/Folk** |`classical.jamulus.io:22524`|
+|**Genre Choral/Barbershop** |`choral.jamulus.io:22724`|
+
+
+#### Running as a Directory
+
+If you wish to run a [Custom Directory](Running-a-Server#3-custom-directory) please see [this guide](Custom-Directories).
+
+### Maintainance
+
+#### Upgrading
+
+To upgrade your Server to a newer version, simply download a new .deb and re-install as step 3.
+
+#### Viewing The Logs
+
+Jamulus will log to the system log file if you left the `StandardOutput=journal` setting in the unit file. 
+
+To view the log, use `journalctl` (to exit press Ctrl-C). For example, to read the system log file, filtered for the Jamulus service:
+
+`journalctl -f -u jamulus-headless`
 
-Note also that mean ADSL2 transfer rate is 10 Mbit/s for downstream and 1 Mbit/s for upstream. The actual performance depends on distance to the provider, which may [theoretically range from 24 Mbit/s at 0.3 km to 1.5 Mbit/s at 5.2 km](https://en.wikipedia.org/wiki/Asymmetric_digital_subscriber_line) for download rate.
+#### Recording
 
+When using the recording function with the `-R` command line option, if the Server receives a SIGUSR1 signal during a recording, it will start a new recording in a new Directory. SIGUSR2 will toggle recording enabled on/off.
 
-# Starting a Server
+To send these signals using systemd, create the following two `.service` files in `/etc/systemd/system`, calling them something appropriate (e.g. `newRecording-Jamulus-serv
+er.service`).
 
+**Note:** You will need to save recordings to a path _outside_ of the jamulus home Directory, or remove `ProtectHome=true` from your systemd unit file (be aware that doing so is however a potential security risk).
+
+For turning recording on or off (depending on the current state):
+
+~~~
+ [Unit]
+ Description=Toggle recording state of Jamulus Server
+ Requisite=Jamulus-Server
+
+ [Service]
+ Type=oneshot
+ ExecStart=/bin/systemctl kill -s SIGUSR2 Jamulus-Server
+~~~
+
+For starting a new recording:
+
+~~~
+ [Unit]
+ Description=Start a new recording on Jamulus Server
+ Requisite=Jamulus-Server
+
+ [Service]
+ Type=oneshot
+ ExecStart=/bin/systemctl kill -s SIGUSR1 Jamulus-Server
+~~~
+
+_Note: The Jamulus service name in the `ExecStart` line needs to be the same as the `.service` file name you created when setting systemd to control your Jamulus Server. So in this example it would be `Jamulus-Server.service`_
+
+Run `sudo systemctl daemon-reload` to register them for first use.
+
+Now you can run these with the `service start` command, for example:
```

Comment:
Shouldn't be using the `service` command, it's deprecated.  `sudo service (something) start` should be replaced with `sudo systemctl start (something)`.